Automattic/woocommerce-payments
# Load subscriptions-core via composer and remove the "includes/subscriptions/subscriptions-base" directory

Fixes #2958 

#### Changes proposed in this Pull Request



Now that Subscriptions Core has been moved to it's own repository](https://github.com/Automattic/woocommerce-subscriptions-core), we no longer need to include `subscriptions-base` inside woocommerce-payments and instead, can load subscriptions-core via composer.

The changes in this PR are pretty straight forward:
 - Update `composer.json` to include the subscriptions-core repo
 - Rename `wcpay_init_subscriptions_base()` to `wcpay_init_subscriptions_core()` and update references to base
 - Update `WCPAY_SUBSCRIPTIONS_ABSPATH` to locate subscriptions from `vendor/`

#### Testing instructions



- Deactivate WooCommerce Subscriptions extension
- run `composer update` to pull subscriptions-core
- Check subscriptions functionality is loaded
  1. Check **WooCommerce > Subscriptions** is there
  2. Check **WooCommerce > Settings > Subscriptions** has limited settings (mixed checkout and syncing)
- Activate WooCommerce Subscriptions and confirm all the premium extension settings are being loaded
